test/actions/navigate-to-challenge.js

```diff
@@ -47,19 +45,14 @@ describe('navigateToChallenge action', () => {
 
       let nextState = reducer(initialState, navigateToChallenge(1, 2));
 
-      expect(nextState).toEqual(initialState.merge({
+      // by starting with nextState and merging in what we expect,
+      // we eliminate all other properties from consideration
+      expect(nextState).toEqual(nextState.merge({
```

Comment:
Good point, but the way it was before I had to fix the test after every unrelated state change, which didn't seem ideal, either. I'll see if I can come up with a better solution.